stream-tests/src/test/resources/Makefile

```diff
@@ -0,0 +1,15 @@
+# Documents how truststore and keystore were created
```

Comment:
do we need a new file? incubator-pekko/remote/src/test/resources/Makefile is very similar to this - can we update that instead?

Reply:
The difference in this case is that there is no SAN extension for stream-tests. I think it's worth highlighting that. Agree this could be done in a shared file but then it would have to go in a logical location / we should use a symlink.

Reply:
can you add a short(-ish) comment to the new file to highlight its similarity to remote/src/test/resources/Makefile ?

Reply:
added